interface/frontend/src/components/common/OpenDICOM.vue

```diff
@@ -1,20 +1,22 @@
 <template>
-  <div class="DICOM-container" style="width: 100%">
```

Comment:
Please don't make unrelated changes in commits - when you use the word "and" or a command (",") in a commit message, it often means that you should split it up into separate commits - makes reviewing and discussion so much easier! (eg. why remove this style? you could explain here but "nobody" will look or can find at this review comment later alas..)

Reply:
@lamby noted! Will update my PR. Btw, this change is a part of fixing the bug with misinterpretation of the predicted coordinates.

Reply:
Moved to separate commit.